<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>candidatoWin</class>
 <widget class="QWidget" name="candidatoWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-5</y>
     <width>951</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resource/img/candidato.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCerrarSesion">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid;
background: #921B63;
color:white;
border-radius: 9px;
font-weight:bold;
font-size:11px;
</string>
   </property>
   <property name="text">
    <string>CERRAR SESIÓN</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNombre">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ricardo Alfonso Hernández Gutierrez</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblEleccion">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>110</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Elecciones Presidenciales México 2019</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVerResultados">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>480</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px solid;
background: #B7387A;
color:white;
border-radius: 15px;
font-weight:bold;
font-size:11px;
</string>
   </property>
   <property name="text">
    <string>VER RESULTADOS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblEstado">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(0,0,0,0.5);</string>
   </property>
   <property name="text">
    <string>Veracruz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Partido:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblPartido">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(0,0,0,0.5);</string>
   </property>
   <property name="text">
    <string>Partido Revolucionario Institucional</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Sus Propuestas:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="lblPropuesta">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background: rgba(0,0,0,0);
color:rgba(0,0,0,0.5);</string>
   </property>
   <property name="plainText">
    <string>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum aliquet felis id justo tempus, nec lacinia sapien vulputate. Quisque erat erat, aliquam nec quam eget, scelerisque ullamcorper lorem. Proin lorem est, elementum vehicula vehicula ut, venenatis feugiat enim. Morbi interdum viverra bibendum. Cras pulvinar augue eu lacus consectetur posuere. Sed hendrerit hendrerit mi non imperdiet. Integer ac bibendum enim, sed vulputate eros. Nulla ullamcorper urna quis quam vestibulum tempus. Sed varius vehicula dui. Aenean hendrerit eros vel efficitur pretium. Integer ultrices eros at mauris iaculis porttitor sed eu nibh. Morbi feugiat nunc eget odio luctus semper. Nulla non tellus tellus. Integer at dolor id purus vestibulum volutpat at ac elit. </string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
